LICENSE.md

```diff
@@ -1,83 +1,79 @@
 ---
-layout: page
 title: "Licenses"
-permalink: /license/
 ---
+
 ## Instructional Material
 
-All Software Carpentry and Data Carpentry instructional material is
+All High Performance Computing Carpentry instructional material is
 made available under the [Creative Commons Attribution
 license][cc-by-human]. The following is a human-readable summary of
 (and not a substitute for) the [full legal text of the CC BY 4.0
 license][cc-by-legal].
 
 You are free:
 
-* to **Share**---copy and redistribute the material in any medium or format
-* to **Adapt**---remix, transform, and build upon the material
+- to **Share**---copy and redistribute the material in any medium or format
```

Comment:
The original asterisks match the item markers under "Notices" while the substituted dashes match those in "Under the following terms." Please harmonize the symbol used for itemized lists throughout each document, if not the entire lesson.

Reply:
Ok, I changed to dashes for all itemized lists, and also restored the original indentation in **No additional restrictions** -- but I don't understand why it is not itemized in the text.